vmdb/app/models/miq_ae_datastore.rb

```diff
@@ -220,4 +224,11 @@ def self.get_domain_object(name, enabled)
     arel = arel.where(:enabled => enabled) unless enabled.nil?
     arel.first
   end
+
+  def self.preserved_attrs_for_domains
+    MiqAeDomain.all.each_with_object({}) do |dom, h|
+      next if dom.name.downcase == MANAGEIQ_DOMAIN.downcase
```

Comment:
Is there a reason we need to skip the ManageIQ domain?  Is this to make sure it always gets set with priority 0 and is locked?